packages/destination-actions/src/destinations/gladly/gladly-operations.ts

```diff
@@ -0,0 +1,105 @@
+import { IntegrationError, RequestClient } from '@segment/actions-core'
+import { mappings } from './gladly-mappings'
+import type { Settings } from './generated-types'
+import type { GenericPayload, Customer } from './gladly-shared-types'
+
+export const API_VERSION = '/api/v1/'
+
+const resources = {
+  createItem: 'customers/{id}/conversation-items',
+  createCustomer: 'customer-profiles',
+  findCustomer: 'customer-profiles?{param}',
+  updateCustomer: 'customer-profiles/{id}'
+}
+
+export class Gladly {
+  url: string
+  request: RequestClient
+
+  constructor(settings: Settings, request: RequestClient) {
+    this.url = settings.url
+    this.request = request
+  }
+
+  createConversationItem = async (customerId: string, payload: GenericPayload) => {
```

Comment:
The signature of this method should be customerId: string, customerItem: CustomerItem